doc/api/package.json.md

```diff
@@ -0,0 +1,307 @@
+# Package.json
+
+<!--introduced_in=REPLACEME-->
+<!-- type=misc -->
+
+## Introduction
+
+A folder containing a `package.json` file, and all subfolders below that folder
+down until the next folder containing another `package.json`, is considered a
+_package scope_. This document aims to describe the fields used by Node.js
+runtime.
+
```

Comment:
It would likely be good to say something in here about how the `package.json` file is used by multiple tools and that additional fields may be used that Node.js does not necessarily support.

Reply:
I think it should also link npm documentation about this file.

Reply:
Agree with both comments above, and point to the [yarn docs](https://classic.yarnpkg.com/en/docs/package-json/ ), too.

Reply:
I don't think node docs should be linking to a package manager it doesn't ship; there's far more than npm or yarn out there (there's more than one set of relevant yarn docs, even - v1 vs v2), and it's a slippery slope to include this.

Reply:
@ljharb Even if we don't deep link to npm and yarn docs, we have to mention that the file is used by other tools, or we will be flooded by bug reports about the package.json fields that we "failed to document". Actual links have the downside of possibly becoming broken, but they are what is most useful to readers of the docs, and I think our docs should stive to be most useful.

Reply:
Absolutely it should mention that fields can be used by other tools; I just don't think node docs should be enumerating those.

Reply:
Ah, so you think we should link to only npm, because we only ship npm?

Reply:
I think we should link to npm's documentation and mention that other tools use `package.json`, and that's it. Npm is special both because Node ships it and that tool either created the `package.json` file/spec or defined most of the fields in it.

Reply:
@sam-github yes.

Reply:
Resolved via 98454d647ff23deaf1d0226c0de7406fc3f2fa91